server/controllers/cohortController.js

```diff
@@ -1,6 +1,15 @@
-export const getAllCohorts = (req, res) => {
-    res.status(200).json("hello from Cohorts route");
-};
+import db from "../db";
+export const getAllCohorts = async (req, res) => {
+    try {
+      const q = "SELECT * FROM cohorts";
+      const result = await db.query(q);
+      const cohorts = result.rows; // every rows has three columns currently.
+      res.status(200).json({ cohorts });
+    } catch (error) {
+      console.error("Error retrieving cohorts:", error);
+      res.status(500).send("Failed, Internal Server Error. Database or table not found");
```

Comment:
Well done on dealing with the error.
You probably dont need to tell the API user about "Database or table not found" as they have no way to change this